### PR TITLE
[DSCP-123] Simple launcher script

### PR DESCRIPTION
**Pull request by Martoon-00**
Added launcher scripts for both educator and witness nodes.
Without that, launching educator is now nontrivial (I want to avoid questions like which password to use or where educator's secret lies). So we need at least an always-actual example of node CLI parameters.  

Layout of files used by nodes is also changed to keep everything in single place. Read-only files are put separately from files which change after every launch, both in dedicated directories (not to litter project root), although I don't think I managed to end with good names for those directories.

### CODE REVIEW

File: scripts/launch/node.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```
Comment (lambda76) [inline scripts/launch/node.sh]: Please use `/usr/bin/env bash`, `/bin/bash` dose not exist on NixOS.

File: scripts/launch/node.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+--db-path $tmp_files/witness.db
+--log-dir $tmp_files/logs
+"
+
+if [[ "$mode" == "educator" ]]; then
+    stack exec "disciplina-educator" -- $witness_params $educator_params
+elif [[ "$mode" == "witness" ]]; then
+    stack exec "disciplina-witness" -- $witness_params
+elif [[ "$mode" == "" ]]; then
+    echo "Please specify which node to run (e.g. \"educator\" or just \"e\")"
```
Comment (lambda76) [inline scripts/launch/node.sh]: `'s/node/mode/'`

File: scripts/launch/node.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+##################
+
+for var in "$@"
+do
+    if [[ "$var" == "witness" ]] || [ "$var" == "w" ]; then
+        mode="witness"
+    elif [[ "$var" == "witness" ]] || [ "$var" == "e" ]; then
+        mode="educator"
+    else
+        echo "Unknown parameter \"$var\", ignoring"
```
Comment (volhovm) [inline scripts/launch/node.sh]: Better fail.

File: scripts/launch/node.sh (hunk 2 above)
Comment (Martoon-00) [reply scripts/launch/node.sh]: That is so for purpose, I see it like we have educator node and witness node.

Comment (Martoon-00) [reply scripts/launch/node.sh]: Right, set it `node` everywhere

**Comment by Martoon-00** [comment]
Thanks for comments, fixed.

**Pull request merged**